tests/async_client_test.py

```diff
@@ -1623,6 +1626,113 @@ async def test_room_create__typed(self, async_client, aioresponse):
+
+        resp = await async_client.room_create(
+            visibility=RoomVisibility.public,
+            alias="foo",
+            name="bar",
+            topic="Foos and bars",
+            room_version="5",
+            preset=RoomPreset.trusted_private_chat,
+            invite={ALICE_ID},
+            initial_state=[],
+            power_level_override={},
+            predecessor={"room_id": TEST_ROOM_ID, "event_id": "event_id_1"},
+        )
+        assert isinstance(resp, RoomCreateResponse)
+        assert resp.room_id == TEST_ROOM_ID2
+
+    async def test_room_create__predecessor_noevent_id(self, async_client, aioresponse):
+        def room_create_cb(url, data, **kwargs):
+            # TODO: Validate `data` completely?
+            if isinstance(data, bytes):
+                data = data.decode(encoding="utf-8")
+            if isinstance(data, str):
+                data = json.loads(data)
+            assert (
+                data.get("creation_content", {})
+                .get("predecessor", {})
+                .get("room_id", "")
+                == TEST_ROOM_ID
+            )
+            assert (
+                data.get("creation_content", {}).get("predecessor", {}).get("event_id")
+                is None
+            )
+            return CallbackResult(
+                status=200, payload=self.room_id_response(TEST_ROOM_ID2)
+            )
+
+        aioresponse.post(f"{BASE_URL_V3}/createRoom" "", callback=room_create_cb)
+
+        resp = await async_client.room_create(
+            visibility=RoomVisibility.public,
+            alias="foo",
+            name="bar",
+            topic="Foos and bars",
+            room_version="5",
+            preset=RoomPreset.trusted_private_chat,
+            invite={ALICE_ID},
+            initial_state=[],
+            power_level_override={},
+            predecessor={"room_id": TEST_ROOM_ID},
+        )
+        assert isinstance(resp, RoomCreateResponse)
+        assert resp.room_id == TEST_ROOM_ID2
+
+    async def test_room_create__additional_creators(self, async_client, aioresponse):
+        def room_create_cb(url, data, **kwargs):
+            # TODO: Validate `data` completely?
+            if isinstance(data, bytes):
+                data = data.decode(encoding="utf-8")
+            if isinstance(data, str):
+                data = json.loads(data)
+            assert data.get("creation_content", {}).get("additional_creators", []) == [
+                ALICE_ID
+            ]
+            return CallbackResult(
+                status=200, payload=self.room_id_response(TEST_ROOM_ID)
+            )
+
+        aioresponse.post(f"{BASE_URL_V3}/createRoom" "", callback=room_create_cb)
+
+        resp = await async_client.room_create(
+            visibility=RoomVisibility.public,
+            alias="foo",
+            name="bar",
+            topic="Foos and bars",
+            room_version="5",
```

Comment:
```suggestion
            room_version="12",
```
since additional_creators is set

Reply:
Fixed.